apps/server/src/shared/domain/entity/school.entity.ts

```diff
@@ -84,15 +88,36 @@ export class School extends BaseEntity {
 
 	constructor(props: ISchoolProperties) {
 		super();
-		if (props.externalId) this.externalId = props.externalId;
-		if (props.inMaintenanceSince) this.inMaintenanceSince = props.inMaintenanceSince;
-		if (props.inUserMigration !== null) this.inUserMigration = props.inUserMigration;
-		if (props.oauthMigrationPossible !== null) this.oauthMigrationPossible = props.oauthMigrationPossible;
-		if (props.oauthMigrationMandatory !== null) this.oauthMigrationMandatory = props.oauthMigrationMandatory;
+		if (props.externalId) {
+			this.externalId = props.externalId;
+		}
+		if (props.inMaintenanceSince) {
+			this.inMaintenanceSince = props.inMaintenanceSince;
+		}
+		if (props.inUserMigration !== null) {
+			this.inUserMigration = props.inUserMigration;
+		}
+		if (props.oauthMigrationPossible !== null) {
```

Comment:
Is null correct here? You set it everywhere to undefined.
You could check everything like this:
if (props.oauthMigrationMandatory) {
  this.oauthMigrationMandatory = props.oauthMigrationMandatory;
}